extras/linked-token/script/ConfigManager.sol

```diff
@@ -33,7 +33,7 @@ contract ConfigManager is Script {
             jsonData = vm.readFile(path);
         } else {
             // If the file doesn't exist, initialize an empty JSON object
-            jsonData = "{\"LinkedToken\":{\"USDCTest\":{}, \"LinkedTokenReplica\":{}, \"LinkedTokenController\":{}}}";
+            jsonData = "{\"LinkedToken\":{\"USDCTest\":{}, \"LinkedTokenReplica\":{}, \"LinkedTokenController\":{}, \"LinkedTokenControllerImplementation\":{}, \"LinkedTokenReplicaImplementation\":{}}}";
```

Comment:
I was hoping to keep the jsonData for LinkedTokenReplica and LinkedTokenController the same and not need to have Proxy suffixes. This would be in line with the proxy being "transparent" and in general to interact with the proxy you would not need to know it is a proxy. I was concerned about having to make many changes in many places while landing a large refactor here. I would prefer this in a follow on PR with the config refactor to simplify landing these changes related to upgrades

Reply:
the suggested change was made here: 59c93df156ef49206d2946cd4cacd2419bb916c1